src/Host/Client/Impl/Host/RHostConnector.cs

```diff
@@ -100,13 +106,12 @@ internal abstract class RHostConnector : IRHostConnector {
             }
         }
 
-        public async Task<RHost> ConnectAsync(string name, IRCallbacks callbacks, string rCommandLineArguments = null, int timeout = 3000, CancellationToken cancellationToken = new CancellationToken()) {
+        public async Task<RHost> ConnectAsync(string name, IRCallbacks callbacks, int timeout = 3000, CancellationToken cancellationToken = new CancellationToken()) {
```

Comment:
Then there is a question how do we want to specify this command line args. Currently they are part of connection settings. If we want one HostConnector to be able to connect with different CommandLine parameters, how user will provide them?

---
In reply to: [75946453](https://github.com/Microsoft/RTVS/pull/2192#discussion_r75946453) [](ancestors = 75946453)